deploy/hackathon-docker/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+#
+# This file is covered by the LICENSING file in the root of this project.
+#
+
+### Dockerfile for open-hacnathon(server and client)
+### based on python official image
+
+FROM python:2.7
+
+USER root
+
+#logs
+RUN mkdir -p /var/log/open-hackathon/
+# server
+RUN mkdir -p /opt/open-hackathon/open-hackathon-server
+ADD ./open-hackathon-server /opt/open-hackathon/open-hackathon-server
+#client
+RUN mkdir -p /opt/open-hackathon/open-hackathon-client
+ADD ./open-hackathon-client /opt/open-hackathon/open-hackathon-client
+#init
+ADD ./deploy/hackathon-docker/docker-entrypoint.sh /opt/open-hackathon
+
+RUN pip install -r /opt/open-hackathon/open-hackathon-server/requirements.txt
+RUN pip install -r /opt/open-hackathon/open-hackathon-client/requirements.txt
+
+ENTRYPOINT ./opt/open-hackathon/docker-entrypoint.sh
```

Comment:
if we run client and server in one container, we can't make sure our client or server process is health even if container still running.

Reply:
we can read log files

Reply:
That's another question.... if Logs are inside container, it might lose if container restarts. We may want to using PV also

Reply:
PV can't solve log problem, We need find another way, 
but in fact, the logs won't be lost when container restarts, unless container was removed(kube will recycle unhealthy container).

And the problem is if the state of the container does not reflect the state of the application，We can't use container restart policy to recover our application